setup.py

```diff
@@ -41,8 +48,8 @@ def get_description():
     url="https://github.com/fedora-infra/fmn.rules",
     download_url="https://pypi.python.org/pypi/fmn.rules/",
     license='LGPLv2+',
-    install_requires=requires,
-    tests_require=tests_require,
+    install_requires=get_requirements('requirements.txt'),
+    tests_require=get_requirements('tests-requirements.txt'),
```

Comment:
nitpick that you can ignore, but I think "test-requirements" sounds more natural than "tests-requirements".